inventory/routes/concepts.rst

```diff
@@ -7,8 +7,7 @@ Concepts
    :glob:
 
    concepts/use_routes
-   concepts/procurement_rule
-   concepts/push_rule
+   concepts/pull_push_rules
```

Comment:
Redirects ?

Reply:
I made the redirects for all the closed PR yesterday.
I will have a look at the open ones today or tomorrow.

Reply:
There is not rush for open PR, don't worry. I'm just leaving comments to make sure that the redirect is added before that the PR is merged.